### PR TITLE
Users/kaadhina/ui changes

### PR DESCRIPTION
**Pull request by kaadhina**
Introducing Run Title field for publish results.
UI changes for other tasks.
[Pending E2E validation through Ant/Maven/Gradle]

### CODE REVIEW

**Comment by allendm-msft** [comment]
looks fine

File: Tasks/ANT/ant.ps1 [hunk 1]
```diff
@@ -119,7 +120,7 @@ if($publishJUnitResultsFromAntBuild)
     else
     {
         Write-Verbose "Calling Publish-TestResults"
-        Publish-TestResults -TestRunner "JUnit" -TestResultsFiles $matchingTestResultsFiles -Context $distributedTaskContext
+        Publish-TestResults -TestRunner "JUnit" -TestResultsFiles $matchingTestResultsFiles -Context $distributedTaskContext -RunTitle $testRunTitle
```
Comment (acesiddhu) [inline Tasks/ANT/ant.ps1]: whats the experience when task is new and agent is old

File: Tasks/RunDistributedTests/task.json [hunk 2]
```diff
@@ -13,25 +13,25 @@
...
         "Minor": 0,
-        "Patch": 10
+        "Patch": 11
     },
     "demands": [
     ],
     "minimumAgentVersion": "1.88.0",
     "groups": [
         {
-            "name": "testMachineGroups",
```
Comment (acesiddhu) [inline Tasks/RunDistributedTests/task.json]: hope u have tested compat

**Comment by acesiddhu** [comment]
I don't see changes for DTA task etc on agent side. assuming u will send one

**Pull request merged**